theory/solutions.js

```diff
@@ -0,0 +1,54 @@
+/* 
+ * Find regexes that match the following. (e.g. find a 
+ * single regex that matches both `antelope` and `antelopes`.)
+
+ * Single regex that matches either of these:
+
+ *    antelope rocks out
+    
+ *    antelopes rock out
+ */
+
+let firstRegex = /antelope(s rock| rocks) out/
+console.log(firstRegex.test('antelope rocks out')) // => true
+console.log(firstRegex.test('antelopes rock out')) // => true
+
+/* Regex that matches either of:
+
+ *    goat
+    
+ *    moat
+
+ * but not:
+
+ *    boat
+ */
+
+let secondRegex = /[gm]oat/
```

Comment:
Nice work! We could also use a capture group if we wanted to match multiple instances instead of the first one found.

```
(g|m)oat
```